bar_industry/data/restaurant_floor.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="pos_restaurant.floor_main" model="restaurant.floor" forcecreate="1">
```

Comment:
Does that change anything from the default from the scenario? If not, I'd remove this file